File: src/main/java/me/arasple/mc/trmenu/action/acts/ActionUnknow.java
package me.arasple.mc.trmenu.action.acts;

import me.arasple.mc.trmenu.action.base.AbstractAction;
import org.bukkit.entity.Player;

/**
 * @author Arasple
 * @date 2020/2/9 15:44
 */
public class ActionUnknow extends AbstractAction {

    @Override
    public String getName() {
        return "unknown";
    }

    @Override
    public void onExecute(Player player) {
        player.sendMessage("§8[§3Tr§bAction§8] §cUNKNOW | §7Input Action: §f" + getContent());
    }

}

File: src/main/kotlin/me/arasple/mc/trmenu/modules/action/impl/ActionUnknow.kt
package me.arasple.mc.trmenu.modules.action.impl

import me.arasple.mc.trmenu.api.Extends.sendLocale
import me.arasple.mc.trmenu.modules.action.base.Action
import org.bukkit.entity.Player

/**
 * @author Arasple
 * @date 2020/3/1 9:39
 */
class ActionUnknow : Action("trmenu_action_unknow") {

    override fun onExecute(player: Player) = player.sendLocale("ERRORS.ACTION", getContent())

}